parser/src/lexer.rs

```diff
@@ -31,7 +31,9 @@ impl<I> StreamOnce for LocatedStream<I>
                 self.location.bump(ch);
                 // HACK: The layout algorithm expects `1` indexing for columns -
                 // this could be altered in the future though
-                self.location.column += Column::from(1);
+                if self.location.column == Column::from(0) {
+                    self.location.column = Column::from(1);
+                }
```

Comment:
@brendanzab  I may have misled you earlier here, just setting the column to 1 is the correct way. Incrementing would get executed on every uncons causing the column to increase twice as fast as it should have 😓